src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/Spinner.xaml.cs

```diff
@@ -31,7 +34,7 @@ internal class EllipseData : ObservableCollection<EllipseDetails>
                 0xE6, 0xCD, 0xB3, 0x9A, 0x80, 0x67, 0x4D, 0x34, 0x1A, 0xFF
             };
 
-        private static readonly SolidColorBrush IndicatorFill = (SolidColorBrush)(new BrushConverter().ConvertFrom("#007ACC"));
+        private readonly SolidColorBrush IndicatorFill = (SolidColorBrush)(new BrushConverter().ConvertFrom("#007ACC"));
```

Comment:
This change is necessary because this WPF element has thread affinity.  This element is created the first time it is accessed, during the first unit test.  At that time the element remembers the current thread as its owning thread.  During execution of subsequent tests on other threads, accessing this element's properties will throw an `InvalidOperationException` because the current thread is different than the element's owning thread.